plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/githubPullRequest.ts

```diff
@@ -298,6 +312,19 @@ export const createPublishGithubPullRequestAction = (
       );
 
       try {
+        const gitAuthorInfo =
+          gitAuthorEmail && gitAuthorName
+            ? { email: gitAuthorEmail, name: gitAuthorName }
+            : undefined;
+
+        const gitCommitterInfo =
+          gitAuthorEmail || gitAuthorName
+            ? {
+                email: gitAuthorEmail ?? gitAuthorEmail,
+                name: gitAuthorName ?? gitAuthorName,
+              }
+            : undefined;
```

Comment:
Please make sure that this follows the same semantics as https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlab.ts#L211C7-L218